tensorflow/buildspec.yml

```diff
@@ -34,36 +34,25 @@
         target: dockerd-entrypoint.py
 
   images:
-    BuildTensorflowCPUTrainPy3DockerImage:
-      <<: *TRAINING_REPOSITORY
-      build: &TENSORFLOW_CPU_TRAINING_PY3 false
-      image_size_baseline: 4899
-      device_type: &DEVICE_TYPE cpu
-      python_version: &DOCKER_PYTHON_VERSION py3
-      tag_python_version: &TAG_PYTHON_VERSION py36
-      tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION ]
-      docker_file: !join [ docker/, *VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile., *DEVICE_TYPE ]
-      context:
-        <<: *TRAINING_CONTEXT
     BuildTensorflowGPUTrainPy3DockerImage:
       <<: *TRAINING_REPOSITORY
       build: &TENSORFLOW_GPU_TRAINING_PY3 false
-      image_size_baseline: 4899
+      image_size_baseline: 7738
       device_type: &DEVICE_TYPE gpu
       python_version: &DOCKER_PYTHON_VERSION py3
-      tag_python_version: &TAG_PYTHON_VERSION py36
+      tag_python_version: &TAG_PYTHON_VERSION py37
       tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *CUDA_VERSION, "-", *OS_VERSION ]
       docker_file: !join [ docker/, *VERSION, /, *DOCKER_PYTHON_VERSION, /, *CUDA_VERSION, /Dockerfile., *DEVICE_TYPE ]
       context:
         <<: *TRAINING_CONTEXT
     BuildTensorflowExampleGPUTrainPy3DockerImage:
       <<: *TRAINING_REPOSITORY
       build: &TENSORFLOW_GPU_TRAINING_PY3 false
-      image_size_baseline: 4899
+      image_size_baseline: 7738
```

Comment:
Why has this changed so much?

Reply:
The current 2.3 cuda10.2 image size is 5.29GB. CUDA11 and cuDNN8 changes - cuda, libnvinfer
cuda upgrade is contributing ~ 1.3G itself.

Reply:
We had this same issue last time. You will need to justify why this increase is happening. It looks like CUDA 11 is being installed on top of cuda 10. We need to find out why. 
Most likely nvinfer, but get that breakdown. Or ask Asimov to give you the list of packages from the new container.

Reply:
I verified that cuda10 is not installed in the docker. I'll get the breakdown.

Reply:
With cuDNN8, NVIDIA has introduced task specific optimizations. Because of this, we see extra lib files like
96M	libcudnn_adv_infer.so.8.0.2
63M	libcudnn_adv_train.so.8.0.2
967M      libcudnn_cnn_infer.so.8.0.2
50M	libcudnn_cnn_train.so.8.0.2
240M      libcudnn_ops_infer.so.8.0.2
25M	libcudnn_ops_train.so.8.0.2
Refer what's New section in https://developer.nvidia.com/cudnn
Also, previosly cuda-10.0 was 1.5G where as cuda-11.0 is 2.8G which is ~1.3G [because of increase in size of libcusolver, libcublas, libcufft libraries.]

To summarize, the size shoot up is because of cnn infer lib and cuda upgrade.